src/components/TipInput.vue

```diff
@@ -152,25 +152,28 @@ export default {
       }
       return null;
     },
-    userProfileDeepLink() {
-      const url = new URL(`${process.env.VUE_APP_WALLET_URL}/tip`);
-      url.searchParams.set('url',
-        `${encodeURIComponent(`https://superhero.com/#/user-profile/${this.userAddress}`)}`);
-      url.searchParams.set('x-success', window.location);
-      url.searchParams.set('x-cancel', window.location);
-      return url;
-    },
     derivedUserTipStats() {
       if (!this.stats || !this.stats.by_url) {
         return null;
       }
       return this.stats.by_url.find((tipStats) => tipStats.url === `https://superhero.com/#/user-profile/${this.userAddress}`);
     },
     deepLink() {
-      const url = new URL(`${process.env.VUE_APP_WALLET_URL}/${this.isRetip ? 'retip' : 'tip'}`);
-      url.searchParams.set('id', this.tip.id);
-      url.searchParams.set('x-success', window.location);
-      url.searchParams.set('x-cancel', window.location);
+      let url = '';
+      if (this.userAddress) {
```

Comment:
this block of ifs could be much simplified (with no else) but could be left as a technical debt